.github/workflows/PR-check.yml

```diff
@@ -46,13 +46,14 @@ jobs:
   pysh-check:
     runs-on: ubuntu-22.04
     steps:
-        - uses: actions/checkout@v3
-        - name: Get scripts internal...
-          run: | 
-                git config --global url."https://${{ secrets.ACCESS_TOKEN }}@github.com/".insteadOf "git@github.com:"
-                git clone git@github.com:PelionIoT/scripts-internal.git
+        - uses: actions/checkout@v4
```

Comment:
the same line twice

Reply:
It's actually not. The 1st one will clone this repo.
The same line closed the scripts-internal repo.